FlowCrypt/Functionality/Api/IdTokenUtils.swift

```diff
@@ -16,10 +16,9 @@ class IdTokenUtils {
         }
 
         let googleAuthManager = GoogleAuthManager(
-            currentUserEmail: userEmail ?? user?.email,
             appDelegateGoogleSessionContainer: nil
```

Comment:
`appDelegateGoogleSessionContainer` has default value `nil` in `GoogleAuthManager` initializer, so we can remove it here